pkg/api/vm/formatter.go

```diff
@@ -113,6 +114,10 @@ func (vf *vmformatter) formatter(request *types.APIRequest, resource *types.RawR
 	if canMigrate(vmi) {
 		resource.AddAction(request, migrate)
 		resource.AddAction(request, findMigratableNodes)
+
+		if canCPUAndMemoryHotplug(vm) {
```

Comment:
When a hotplug is onging, how to avoid re-trigger hotplug;  like vm migration

Reply:
Add two condition to check whether VM can do cpu / memory hotplug.

* HotVCPUChange
* HotMemoryChange